tests/integration/v1/auth/signup.spec.js

```diff
@@ -44,6 +44,20 @@ describe("signup", () => {
       email: 'user@example.com',
     })
 	})
+	it("should not create new user", async () => {
+		await supertest(app).post("/api/v1/auth/signup").send({
+			email: "user@example.com",
+			password: "password"
+		});
```

Comment:
Doesn't this part shows an error, since `user@example.com` should already exists in database from preview test.

Reply:
ok, i will change it user1 instead.